spvc/README.asciidoc

```diff
@@ -0,0 +1,60 @@
+= spvc Manual
+:toc:
+:toclevels: 3
+:numbered:
+:source-highlighter: pygments
+
+== Name
+
+`spvc` - A command-line SPIR-V to GLSL/HLSL/MSL compiler with
```

Comment:
I think there is no way to turn off validation.
And always-on validation was a key motivation for spvc in the first place.  So I'd like to put that up front. 

Suggest:

     spvc  - A validating SPIR-V to GLSL/HLSL/MSL compiler with Clang-compatible options

Reply:
done.